File: DoItTest/ClientApp/src/domain/tests/items/anserOption/textFildAnserOption.ts
import { AnserOption as AnswerOption } from "./anserOption"

export class TextFildAnserOption extends AnswerOption {
    constructor(
        public id: string,
        public testItemId: string,
        public answer: string
    ) {
        super(id, testItemId)
    }
}

export function mapTextFildAnserOption(value: any): TextFildAnserOption {
    return new TextFildAnserOption(value.id, value.testItemId, value.answer)
}

File: DoItTest.Domain/Tests/TestItems/AnswerOptions/TextFildAnserOption.cs
namespace DoItTest.Domain.Tests.TestItems.AnswerOptions
{
    public class TextFildAnserOption : AnswerOption
    {
        public String Answer { get; }

        public TextFildAnserOption(Guid id, Guid testItemId, String answer) : base(id, testItemId)
        {
            Answer = answer;
        }
    }
}
